nemo_curator/modules/fuzzy_dedup/lsh.py

```diff
@@ -272,6 +272,12 @@ def _write_bucket_parquet(
         return wrote_buckets, are_buckets_empty
 
     def __call__(self, dataset: DocumentDataset) -> DocumentDataset:
+        if "cudf" not in str(type(dataset.df)):
```

Comment:
Since this is a public class with a `__call__` method can default to inheriting from BaseModule instead.